resources/builder.inc.sh

```diff
@@ -143,17 +147,47 @@ function builder_heading() {
 #
 ####################################################################################
 
+
+builder_echo() {
+  local color=white message= mark=
+  if [[ $# -gt 1 ]]; then
+    color="$1"
+    message="$2"
+  else
+    message="$1"
+  fi
+
+  if [[ ! -z ${COLOR_RED+x} ]]; then
+    case $color in
+      white) color="$COLOR_WHITE" ;;
+      grey) color="$COLOR_GREY" ;;
+      green|success) color="$COLOR_GREEN" ;;
+      blue|heading) color="$COLOR_BLUE" ;;
+      yellow|warning) color="$COLOR_YELLOW" ;;
+      red|error) color="$COLOR_RED" ;;
+      teal) color="$COLOR_TEAL" ;;
+      setmark) mark="$HEADING_SETMARK" color="$COLOR_PURPLE" ;;
```

Comment:
What about bright white and purple (separate from setmark)?

Reply:
Can do